test/local/routes/sms.js

```diff
@@ -589,3 +589,103 @@ describe('/sms/status with disabled geo-ip lookup', () => {
   })
 })
 
+describe('/sms/status with query param and enabled geo-ip lookup', () => {
```

Comment:
> I know some people prefer a comprehensive/completionist approach though, were you asking from that perspective? 

I guess it's the difference between code coverage and path coverage. Both are reasonable to strive for, though path coverage is far more difficult to attain for complex functions.

Someone is almost certainly going to try to hit the endpoint with a bunk country, what happens in that situation, will there be a 400 error and blocked with joi validation, or will this code be exercised? If this code is exercised, it'd be good to be confident it doesn't blow up with bunk countries.

The front end allows a specific list of countries and will error if a country isn't on the list, though that'll only mitigate potential problems for people that use the front end.